docker-compose.yml

```diff
@@ -5,30 +5,41 @@ services:
     image: front-end-monorepo_dev:latest
     build:
       context: ./
-      target: bootstrap
     entrypoint:
       - "/bin/bash"
   fe-project:
-    image: front-end-monorepo_prod:latest
+    image: front-end-monorepo_dev:latest
     build:
       context: ./
-      target: production-apps
     entrypoint:
       - "yarn"
       - "workspace"
       - "@zooniverse/fe-project"
     command: ["start"]
+    environment:
+      - PORT=3000
+      - ASSET_PREFIX=http://localhost:3000
```

Comment:
does this need to be renamed to match the project / content-page apps context?

Reply:
Not unless we add a proxy to the local setup. I don't think anyone is using different hostnames for local work (couldbe wrong.)

Reply:
Oh wait, you mean this should be `PROJECT_ASSET_PREFIX` now. Yep.

I was thinking about the hostnames for projects vs. content pages, which are different in our production deploys